docs/source/initialsetup/compilingmame.rst

```diff
@@ -290,47 +304,57 @@ Emscripten Javascript and HTML
 ------------------------------
 
 First, download and install Emscripten 1.37.29 or later by following the
-instructions at the `official site <https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html>`_
+instructions at the `official site <https://kripken.github.io/emscripten-site/docs/getting_started/downloads.html>`_.
 
 Once Emscripten has been installed, it should be possible to compile MAME
-out-of-the-box using Emscripten’s **emmake** tool. Because a full MAME compile
-is too large to load into a web browser at once, you will want to use the
-**SOURCES** parameter to compile only a subset of the project, e.g. (in the mame
-directory):
+out-of-the-box using Emscripten's '**emmake**' tool. Because a full MAME
+compile is too large to load into a web browser at once, you will want to use
+the SOURCES parameter to compile only a subset of the project, e.g. (in the
+MAME directory):
 
 .. code-block:: bash
 
-    emmake make SUBTARGET=pacmantest SOURCES=src/mame/drivers/pacman.cpp
+The SOURCES parameter should have the path to at least one driver .cpp file. The
+make process will attempt to locate and include all dependencies necessary to
+produce a complete build including the specified driver(s). However, sometimes
+it is necessary to manually specify additional files (using commas) if this
+process misses something. E.g.:
```

Comment:
This appears to be a duplicate of the following paragraph, with some formatting removed, inadvertently replacing an example command.

Reply:
Quite possibly from the merge going sour. Brought back the corrected example. Thanks.